indy_node/test/upgrade/test_forced_upgrade_no_consensus_on_single_node.py

```diff
@@ -0,0 +1,27 @@
+from indy_node.test import waits
+from indy_node.test.upgrade.helper import sendUpgrade, bumpVersion
+from plenum.common.constants import VERSION
+from stp_core.loop.eventually import eventually
+
+
+def testForcedUpgradeNoConsensusOnSingleNode(
+        validUpgradeExpForceTrue, looper, nodeSet, trustee, trusteeWallet):
+    nup = validUpgradeExpForceTrue.copy()
+    nup.update({VERSION: bumpVersion(validUpgradeExpForceTrue[VERSION])})
+    for node in nodeSet:
+        if node.name != "Alpha":
+            node.cleanupOnStopping = False
+            looper.removeProdable(node)
+            node.stop()
+        else:
+            node.upgrader.scheduledUpgrade = None
+    sendUpgrade(trustee, trusteeWallet, nup)
+
+    def testsched():
+        for node in nodeSet:
+            if node.name == "Alpha":
+                assert node.upgrader.scheduledUpgrade
+                assert node.upgrader.scheduledUpgrade[0] == nup[VERSION]
```

Comment:
Didn't we bump the version, so it should be more than nup[VERSION]?

Reply:
This test was written earlier. In scope of this pull request it is only moved to a separate file.
As I can see, we verify that the target version of the scheduled upgrade is the same as we send in `sendUpgrade` call.